go/enclave/db/storage.go

```diff
@@ -43,14 +47,30 @@ type storageImpl struct {
 }
 
 func NewStorage(backingDB *sql.EnclaveDB, chainConfig *params.ChainConfig, logger gethlog.Logger) Storage {
+	cacheConfig := &gethcore.CacheConfig{
+		TrieCleanLimit: 256,
+		TrieDirtyLimit: 256,
+		TrieTimeLimit:  5 * time.Minute,
+		SnapshotLimit:  256,
```

Comment:
we don't use snapshots do we? But 'm guessing these settings aren't doing any harm?

Reply:
yeah. not used for now